pir/pir-internal/src/main/java/com/duckduckgo/pir/internal/PirRemoteFeatures.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.pir.internal
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.Toggle
+import com.duckduckgo.feature.toggles.api.Toggle.DefaultValue
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    featureName = "pir",
+)
+interface PirRemoteFeatures {
+    @Toggle.DefaultValue(true)
+    fun self(): Toggle
+
+    @DefaultValue(false)
+    fun allowPirRun(): Toggle // kill switch
```

Comment:
Do you want to slap the `InternalAlwaysEnabled` annotation in to ensure all internal users have this running?
Or we want opt-in for internal.